src/components/Connect.vue

```diff
@@ -198,22 +198,18 @@ onUnmounted(() => {
   background-size: contain;
   position: relative;
   top: 3px;
-
```

Comment:
Let's keep separation lines, we have `rule-empty-line-before` in stylelint, looks like our linter is not working as expected. Can you check what is the problem?

Reply:
I actually ran npm run lint:css and didn't get a warning, I'll format accordingly